src/control/server/storage/scm/provider.go

```diff
@@ -611,6 +611,9 @@ func (p *Provider) MountRamdisk(target string, params *storage.RamdiskParams) (*
 	if params.Size > 0 {
 		opts = append(opts, fmt.Sprintf("size=%dg", params.Size))
 	}
+	if !params.DisableHugepages {
+		opts = append(opts, "huge=always")
```

Comment:
Shall we use "mpol=local" too?

Reply:
In a prior commit, (line 609 above this), I added support for `mpol=prefer:%d` where we prefer to allocate memory on the numa node being used by the engine. I don't feel strongly either way, but based on https://man7.org/linux/man-pages/man2/set_mempolicy.2.html it appears that local is the default policy anyhow. I didn't make that configurable, but it wouldn't be difficult to expose that as a tunable so that the effect of different memory policies can be tested.